ttps/macOS/README.md

```diff
@@ -0,0 +1,181 @@
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/clipdump_cli/clipdump_cli.yaml
+```
+
+4. Dump clipboard contents using NSPasteboard API calls.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/clipdump_api/clipdump_api.yaml
+```
+
+5. Search for ssh and cloud credentials on disk.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/keysearch/keysearch.yaml
+```
+
+6. Set up launch agent persistence.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/launchagent/launchagent.yaml
+```
+
+7. Set up login item persistence.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/loginitem/loginitem.yaml
+```
+
+8. Perform a dylib injection.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/injectdylib/injectdylib.yaml
+```
+
+9. Copy the user's login keychain to temp.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/copykeychain/copykeychain.yaml
+```
+
+10. Local network host enumeration.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/service_discovery/service_discovery.yaml
+```
+
+11. Gather system information.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/system_info/system_info.yaml
+```
+
+12. Execute the SwiftBelt system enumerator.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/swiftbelt_exec/swiftbelt_exec.yaml
+```
+
+13. Set up .zshrc persistence.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/zshrc_persist/zshrc_persist.yaml
+```
+
+14. Stealthy TCC Folder Access Check (via API calls).
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/tcc_access_check_api/check.yaml
+```
+
+15. Dylib Execution.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/run_dylib/rundylib.yaml
+```
+
+16. Read browser (Firefox) cookies.
+
+**Pre-requisite: Firefox is installed**
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/dump_cookies/dump_cookies.yaml
+```
+
+17. Check if the screen is locked.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/lockscreen_check/lockscreen_check.yaml
+```
+
+18. Remove the com.apple.quarantine attribute.
+
+- From the root ForgeArmory directory, run: 
+```
+ttpforge run -c ttpforge-repo-config.yaml ttps/macOS/remove_quarantine_attrib/remove_attrib.yaml
```

Comment:
explicitly specifying the config shouldn't be necessary anymore (latest ttpforge should look it up by walking up the filesystem)  - mind checking if it works for you with the `-c ttpforge-repo-config.yaml` removed?